backend/pyproject.toml

```diff
@@ -42,9 +42,11 @@ bump-pydantic = "^0.8.0"
 black = "^24.4.2"
 boto3 = "^1.35.26"
 typing-extensions = "^4.12.2"
+pytest-asyncio = "^0.24.0"
 
 [tool.poetry.dev-dependencies]
-pytest = "^7.2.1"
+faker = "^21.0.0"
```

Comment:
We recently deleted faker as it was breaking the deployed project, why are we adding it back? I don't see it used in your MR.

Reply:
removed